operations/iaas/metering/meter_configuration.md

```diff
@@ -0,0 +1,731 @@
+  traits:
+    <<: *trove_base_traits
+    display_name:
+      fields: payload.display_name
+    audit_period_beginning:
+      type: datetime
+      fields: payload.audit_period_beginning
+    audit_period_ending:
+      type: datetime
+      fields: payload.audit_period_ending
+- event_type: profiler.*
+  traits:
+    project:
+      fields: payload.project
+    service:
+      fields: payload.service
+    name:
+      fields: payload.name
+    base_id:
+      fields: payload.base_id
+    trace_id:
+      fields: payload.trace_id
+    parent_id:
+      fields: payload.parent_id
+    timestamp:
+      type: datetime
+      fields: payload.timestamp
+    host:
+      fields: payload.info.host
+    path:
+      fields: payload.info.request.path
+    query:
+      fields: payload.info.request.query
+    method:
+      fields: payload.info.request.method
+    scheme:
+      fields: payload.info.request.scheme
+    db.statement:
+      fields: payload.info.db.statement
+    db.params:
+      fields: payload.info.db.params
+- event_type: 'magnum.cluster.*'
+  traits: &magnum_cluster_crud
+    id:
+      fields: payload.id
+    typeURI:
+      fields: payload.typeURI
+    eventType:
+      fields: payload.eventType
+    eventTime:
+      type: datetime
+      fields: payload.eventTime
+    action:
+      fields: payload.action
+    outcome:
+      fields: payload.outcome
+    initiator_id:
+      fields: payload.initiator.id
+    initiator_typeURI:
+      fields: payload.initiator.typeURI
+    initiator_name:
+      fields: payload.initiator.name
+    initiator_host_agent:
+      fields: payload.initiator.host.agent
+    initiator_host_address:
+      fields: payload.initiator.host.address
+    target_id:
+      fields: payload.target.id
+    target_typeURI:
+      fields: payload.target.typeURI
+    observer_id:
+      fields: payload.observer.id
+    observer_typeURI:
+      fields: payload.observer.typeURI
+- event_type: 'alarm.*'
+  traits:
+    id:
+      fields: payload.alarm_id
+    user_id:
+      fields: payload.user_id
+    project_id:
+      fields: payload.project_id
+    on_behalf_of:
+      fields: payload.on_behalf_of
+    severity:
+      fields: payload.severity
+    detail:
+      fields: payload.detail
+    type:
+      fields: payload.type
+
+```
+
+The event_pipeline can be used to filter events and pipeline them to different publishers.
+the notifier publisher is the message queue broadcasster (RabbitMQ).
+
+### 1.3 event_pipeline.yaml
+
+```yaml
+---
+sources:
+    - name: event_source # 'source name'
+      events:
+          - "*" # 'event filter'
+      sinks:
+          - event_sink # 'sink name'
+sinks:
+    - name: event_sink # 'sink name'
+      publishers: # 'list of publishers'
+          - notifier://
+
+```
+
+The pipeline.yaml can be used to filter and pipeline all metrics and events since events send via the notifier also appeas in the meters and send them to different publishers.
```

Comment:
```suggestion
The pipeline.yaml can be used to filter and pipeline all metrics and events since events send via the notifier also appears in the meters and send them to different publishers.
```

Reply:
Changed that. Many thanks!